src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteFooter.scss

```
@@ -40,15 +33,59 @@
 }
 
 .note-dialog__note-author-name {
-  margin-left: $margin--default;
+  margin-left: $margin--small;
   color: $color-dark-gray;
   font-size: $text-size--small;
   letter-spacing: $letter-spacing--small;
   line-height: $line-height--small;
 }
 
-.note-dialog .note-dialog__note-user-avatar {
-  height: 34px;
-  width: 34px;
+.note-dialog__note-user-avatar {
+  height: var(--note-footer-height);
+  width: var(--note-footer-height);
   padding: 0;
+  margin-bottom: 4px;
+
+  #Circle-Background {
+    stroke: none;
+  }
+}
+
+.note-dialog__note-author {
+  display: inline-flex;
+  align-items: center;
+  // reset css from figure-element:
+  margin: 0;
+  border: 0;
+  font: inherit;
+  vertical-align: baseline;
+  flex-grow: 0;
+  height: 35px;
+  padding: 0 $margin--default 0 0;
+}
+
+.note-dialog__note-author--self {
+  background: var(--accent-color--desaturated-light);
+  border-radius: 21px;
```

```suggestion
  border-radius: $note__border-radius;
```
From the style.scss:

```scss
// border-radius
$note__border-radius: 16px;
```

---

The radius here is chosen specifically to fit the radius of the avatar